src/test/java/ir/sahab/uncaughtexceptionrule/UncaughtExceptionExtensionTest.java

```diff
@@ -0,0 +1,54 @@
+package ir.sahab.uncaughtexceptionrule;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(UncaughtExceptionExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class UncaughtExceptionExtensionTest {
+
+    @Test
```

Comment:
You didn't test the actual behavior of UncaughtExceptionExtension. Please add one more test and check real behavior of UncaughtExceptionExtension.

Reply:
I've changed it to match the previous test.